src/utils/logger.ts

```diff
@@ -18,7 +18,7 @@ if (config.nodeEnv !== 'dev' && config.nodeEnv !== 'test') {
   });
 }
 
-const logger = config.nodeEnv === 'dev' || config.nodeEnv === 'test' ? console : log4js.getLogger();
+const logger = console;
```

Comment:
Can you revert this line? Otherwise APM won't work

Reply:
Thanks, Joey. This what the problem all along. 👍